src/shared/defaultAwsContextCommands.ts

```diff
@@ -190,43 +197,38 @@ export class DefaultAWSContextCommands {
                 return state.credentialProfile.label
             }
         } else if (credentialsFiles.length === 0) {
-            const userResponse = await window.showInformationMessage(
-                localize(
-                    'AWS.message.prompt.credentials.create',
-                    'You do not appear to have any {0} Credentials defined. Would you like to set one up now?',
-                    getIdeProperties().company
-                ),
-                localizedText.yes,
-                localizedText.no
-            )
-
-            if (userResponse !== localizedText.yes) {
-                return undefined
+            if (await this.promptCredentialsSetup()) {
+                return await this.promptAndCreateNewCredentialsFile()
             }
-
-            return await this.promptAndCreateNewCredentialsFile()
         } else {
-            // If no credentials were found, the user should be
-            // encouraged to define some.
-            const userResponse = await window.showInformationMessage(
-                localize(
-                    'AWS.message.prompt.credentials.create',
-                    'You do not appear to have any {0} Credentials defined. Would you like to set one up now?',
-                    getIdeProperties().company
-                ),
-                localizedText.yes,
-                localizedText.no
-            )
-
-            if (userResponse === localizedText.yes) {
-                // Start edit, the user will have to try connecting again
-                // after they have made their edits.
+            if (await this.promptCredentialsSetup()) {
                 await this.editCredentials()
             }
+        }
+        return undefined
+    }
 
-            // If we get here, there are credentials for the user to choose from
-            return undefined
+    private async promptCredentialsSetup(): Promise<boolean> {
+        // If no credentials were found, the user should be
+        // encouraged to define some.
+        const userResponse = await window.showInformationMessage(
+            localize(
+                'AWS.message.prompt.credentials.create',
+                'You do not appear to have any {0} Credentials defined. Would you like to set one up now?',
+                getIdeProperties().company
+            ),
+            localizedText.yes,
+            localizedText.no,
+            localizedText.help
+        )
+
+        if (userResponse === localizedText.yes) {
+            return true
+        } else if (userResponse === localizedText.help) {
```

Comment:
Does this close the notification? Doesn't do a lot of good if the user clicks this and can't find their way back.

Reply:
Does not close it. Flow controller was modified.

Reply:
Updated this logic to reprompt if they hit help so for all intents and purposes they return to what appears to be the original prompt. Kind of a goofy way to do it but VSCode does not allow onClick event handling that does not also close the prompt.